src/integrationTest/java/com/hivemq/extensions/helloworld/HelloWorldInterceptorIT.java

```diff
@@ -19,7 +19,7 @@
 import com.hivemq.client.mqtt.mqtt5.Mqtt5BlockingClient;
 import com.hivemq.client.mqtt.mqtt5.Mqtt5Client;
 import com.hivemq.client.mqtt.mqtt5.message.publish.Mqtt5Publish;
-import com.hivemq.extension.sdk.api.annotations.NotNull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
curious, why are we not using the hivemq annotations? Is it because it is just a hello world repo?

Reply:
because the integration tests now do not depend on the main dependencies anymore, so there is also no hivemq-extension-sdk available. I did not add a dependency on the hivemq-extension-sdk for only using 2 annotations then.